icu4c/source/test/intltest/unitstest.cpp

```diff
@@ -458,9 +501,53 @@ class ExpectedOutput {
     }
 };
 
+void checkOutput(UnitsTest *unitsTest, const char *msg, ExpectedOutput expected,
```

Comment:
It's not clear to me what this function is testing.  Please add a sentence or two in a comment explaining.

Reply:
I have added a TODO to @hugovdm